requirements.txt

```diff
@@ -0,0 +1 @@
+pyserial
```

Comment:
What is pyserial needed for?

Reply:
If I don't install pyserial, I get an exception on line 15, tiflash-->utils-->detect.py 
(from serial.tools import list_ports)